blog/2019-06-03-debezium-newsletter-01-2019.adoc

```diff
@@ -0,0 +1,48 @@
+= Debezium's Newsletter 01/2019
+ccranfor
+:awestruct-tags: [ community, news, newsletter ]
+:awestruct-layout: blog-post
+
+Welcome to the Debezium community newsletter in which we share blog posts, group discussions, as well as StackOverflow
+questions that are relevant to our user community.
+
+== Articles
+
+Debezium allows replicating data between heterogenious data stores with ease.  https://blog.couchbase.com/kafka-connect-mysql-couchbase-debezium/[This article]
+explains how you can replicate data from MySQL to CouchBase.
+
+As the size of data that systems maintain continues to grow, this begins to impact how we capture, compute, and report
+real-time analytics. https://medium.com/high-alpha/data-stream-processing-for-newbies-with-kafka-ksql-and-postgres-c30309cfaaf8[This article]
+explains how you can use Debezium to stream data from Postgres, perform analytical calculations using KSQL, and then
+stream those results back to Postgres for consumption.
+
+In a recent https://medium.com/@singaretti/streaming-de-dados-do-postgresql-utilizando-kafka-debezium-v2-d49f46d70b37[recent article],
+the author illustrates how they use Debezium and Kafka to stream changes from their relational database and then store
+the change stream results in Goodle Cloud Services.
+
+This https://debezium.io/blog/2019/05/23/tutorial-using-debezium-connectors-with-apache-pulsar/[recent blog] provides
+a complete tutorial showing how to use Debezium connectors with Apache Pulsar.
+
+== Time to upgrade
+
+Debezium version https://debezium.io/blog/2019/05/06/debezium-0-9-5-final-released/[0.9.5] was just released.
+If you are using the 0.9 branch you should definitely check out 0.9.5.  For details on the bug fixes as well as
+the enhancements this version includes, check out the
+https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12317320&version=12341657[release notes].
+
+The Debezium team has also begun active development on the next major version, 0.10.  We recently published
+a https://debezium.io/blog/2019/05/29/debezium-0-10-0-alpha1-released/[blog] that provides an overview
+behind what 0.10 is meant to deliver.  If you want details on the bug fixes and enhancements we've packed
```

Comment:
It's a bit confusing that it speaks of _one_ release, but then two Alphas are linked. Not sure how it could be reworded, but perhaps you have an idea.